### PR TITLE
Fix #8379 [Bug] Send Bug Report - Can't re-submit if cancelled

### LINKED ISSUE #8379: [Bug] Send Bug Report - Can't re-submit if cancelled
###### Reproduction Steps

1. Help - Get Help - Send troubleshooting report
2. Cancel

###### Expected Result

* I can send a report again

###### Actual Result

* I need to wait

###### Debug info
```
AnkiDroid Version = 2.15alpha34

Android Version = 11

Manufacturer = Google

Model = Pixel 3a

Hardware = sargo

Webview User Agent = Mozilla/5.0 (Linux; Android 11; Pixel 3a Build/RQ2A.210305.006; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/89.0.4389.105 Mobile Safari/537.36

ACRA UUID = 2f93c96f-11ae-45bd-b314-b1236df15be0

Scheduler = std2

Crash Reports Enabled = false

DatabaseV2 Enabled = true
```

###### Research
- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by Prashant830**
## Purpose / Description
[Bug] Send Bug Report - Can't re-submit if cancelled

## Fixes
Fixes #8379 

## Approach
fix bug at exceptionReportItem method at HelpDialog.java class.

## How Has This Been Tested?
|               | |
|---------------|-|
|Device         |Infinix hot 7 |
|Android Version| Android 9 |

## Short Video

https://user-images.githubusercontent.com/72181295/113421678-260d4900-93e9-11eb-8a12-48a6c63bf080.mp4

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.java [hunk 1]
```diff
@@ -266,12 +271,12 @@ protected void onClicked(AnkiActivity activity) {
...
 
         private void sendReport(AnkiActivity activity) {
-            if (lastClickStamp == null || currentTimestamp - lastClickStamp > minIntervalMS) {
+            if (lastClickTimeStamp == null || currentTimestamp - lastClickTimeStamp > minIntervalMS) {
                 AnkiDroidApp.deleteACRALimiterData(activity);
                 AnkiDroidApp.sendExceptionReport(
                         new UserSubmittedException(exceptionMessage),
                         "AnkiDroidApp.HelpDialog");
-                lastClickStamp = currentTimestamp;
+                lastClickMap.put(lastClick, currentTimestamp);
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.java:279]: You're not setting `lastClickTimeStamp` any more - this removes rate limiting on the dialog

File: AnkiDroid/src/main/res/drawable/ic_baseline_lock_24.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/drawable/ic_baseline_lock_24.xml:3]: These changes aren't relevant, could you remove them from the PR via a rebase

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Thanks for the PR! This solves the problem by removing the rate limiting, which isn't what we're after

After the problem is solved, could you confirm that the issue is fixed by following these steps:

* Send exception report with preference set to "ask"
* Press "cancel" on the report
* Send exception report
* The dialog should be shown again

**Comment by mikehardy** [comment]
Closing with preference for #8629 to get this done for 2.15

**Pull request closed without merging**